src/Symfony/Component/Serializer/Normalizer/GetSetMethodNormalizer.php

```diff
@@ -47,7 +47,7 @@ public function denormalize($data, $class, $format = null, array $context = arra
         $normalizedData = $this->prepareForDenormalization($data);
 
         $reflectionClass = new \ReflectionClass($class);
-        $object = $this->instantiateObject($normalizedData, $class, $context, $reflectionClass, $allowedAttributes);
+        $object = $this->instantiateComplexObject($normalizedData, $class, $context, $reflectionClass, $allowedAttributes, $format);
```

Comment:
Changing the class name looks like a no go for me...

@Ener-Getick changing the signature of a `propected` method is a BC break isn't it? (not sure of how PHP handle the inheritance of the splat operator).